src/pages/project/AdminProject.js

```diff
@@ -478,7 +478,7 @@ export default function AdminProject({ requestsRoute }) {
                   Are you sure you want to edit this product?
                   <Button
                     onClick={submitForm}
-                    disabled={!(formik.dirty && readOnlyAdminIsAbleToEdit)}
+                    disabled={readOnlyAdminIsAbleToEdit ? !formik.dirty: !readOnlyAdminIsAbleToEdit }
```

Comment:
Instead of using a ternary operator, I think this is more clear and equivalent: `!readOnlyAdminIsAbleToEdit || !formik.dirty`.